tests/unittests/tests-cbor/tests-cbor.c

```diff
@@ -796,23 +796,23 @@ void test_stream_decode(void)
     cbor_serialize_byte_string(&stream, "11");
     cbor_write_break(&stream);
 
-#ifndef CBOR_NO_SEMANTIC_TAGGING
-#ifndef CBOR_NO_CTIME
+#ifdef MODULE_CBOR_SEMANTIC_TAGGING
+#ifdef MODULE_CBOR_CTIME
     time_t rawtime;
     time(&rawtime);
     struct tm *timeinfo = localtime(&rawtime);
     cbor_serialize_date_time(&stream, timeinfo);
     cbor_serialize_date_time_epoch(&stream, rawtime);
-#endif /* CBOR_NO_CTIME */
+#endif /* MODULE_CBOR_CTIME */
 
     /* decoder should skip the tag and print 'unsupported' here */
     cbor_write_tag(&stream, 2);
     cbor_serialize_byte_string(&stream, "1");
-#endif /* CBOR_NO_SEMANTIC_TAGGING */
+#endif /* MODULE_CBOR_SEMANTIC_TAGGING */
 
     cbor_stream_decode(&stream);
 }
-#endif /* CBOR_NO_PRINT */
+#endif /* MODULE_CBOR_PRINT */
```

Comment:
Ah, since this is used here (in the unittests) I would prefer to keep the module to not "poison" the normal unittest output in normal configuration